### DISCUSSION TITLE
Request for Angles manual quickstart instructions

### DISCUSSION THREAD

**Discussion by zhenya-kozhevnikov** [AnglesHQ/angles#68] [General]
Hello! I'm very interested in Angles, but I would like to make basic test of its workflow using manual API requests (creating team, component, environment etc and submit the screenshots).
I couldn't achieve it so far.
May I ask for some kind of short guide on that, similar to one from [angles-webdriverio-example](https://github.com/AnglesHQ/webdriverio-example)? Like a brief step-by-step of how to upload baseline screenshot and next screenshot that will be compared against the baseline,

**Comment by snevesbarros** [comment]
Hey @zhenya-kozhevnikov thanks :). Have you had a look at https://angleshq.github.io/? It describes how to set it up (locally). Once it's up and running  (using docker-compose), you access the swagger docs (by loading e.g. http://127.0.0.1:3000/api-docs) it will give you examples of how to create a team (including components), environment etc. With regards to the screenshot baseline tests, did you want to do this manually or use an automation framework? There are two clients (Java and Javascripts) and as you mentioned an example for webdriverio is already provided.  If helpful I can probably provide some curl requests (I think the swagger api docs also provide that)?
